### PR TITLE
Add WNP 105 for EU (Fixes #12075)

### LINKED ISSUE #12075: Firefox 105 WNP for EU
## Description

Launch date: Sept 20th

- FR: Version 1 (Productivity/Save)
- UK: A/B Test Version 1 and 3 (Save against Discovery)
- DE: A/B Test Version 1 and 2 (Save against KOL’s reasons to use Pocket)

Brief (Mozilla access only): https://docs.google.com/document/d/1OYf-7GVs6_NvCtAMsF0vWHoWJnNUtl0EY4tn0-0utxU/edit#

---

#### Success Criteria
- [ ] WNP is live in production along with A/B test

### PR DESCRIPTION
**Pull request by alexgibson**
## One-line summary

Adds WNP 105 for DE / FR / UK

## Issue / Bugzilla link

#12075

## Testing

Note: both the UK and DE versions have a traffic cop experiment with a 50/50 split across 2 variations.

- http://localhost:8000/en-US/firefox/105.0/whatsnew/?geo=gb
- http://localhost:8000/en-GB/firefox/105.0/whatsnew/?geo=gb
- http://localhost:8000/en-US/firefox/105.0/whatsnew/?geo=gb&v=2
- http://localhost:8000/de/firefox/105.0/whatsnew/
- http://localhost:8000/de/firefox/105.0/whatsnew/?v=2
- http://localhost:8000/fr/firefox/105.0/whatsnew/

Verify traffic cop redirects as expected on:

- [ ] http://localhost:8000/en-US/firefox/105.0/whatsnew/?geo=gb
- [ ] http://localhost:8000/de/firefox/105.0/whatsnew/

### CODE REVIEW

**Comment by alexgibson** [comment]
The German design variation is still awaiting final approvals, but I think this is now code complete enough to move into review.

File: media/css/firefox/whatsnew/whatsnew-105-eu.scss
Comment (maureenlholland) [inline media/css/firefox/whatsnew/whatsnew-105-eu.scss:92]: **issue (blocking)**: longer lockup wordmark is cut off on mobile

**suggestion**: since it's probably still too big when scaled down, might be best to use simple pocket logo until medium or larger screens

Screen Shot 2022-09-12 at 2 01 17 PM

File: bedrock/firefox/templates/firefox/whatsnew/whatsnew-fx105-uk-2.html
Comment (maureenlholland) [inline bedrock/firefox/templates/firefox/whatsnew/whatsnew-fx105-uk-2.html:52]: **style nit (non-blocking)**: one hanging word on large screen headline

**suggestion**: use of `br` tag as in `-fr` template, or maybe a little extra right padding on large screen headline to ensure "inspires" bumps to second line

Screen Shot 2022-09-12 at 2 05 54 PM

**Review by maureenlholland — CHANGES_REQUESTED** [review]
This is amazing work. 5 templates, 3 languages, 2 experiments, lots of custom colors and design elements. Extremely clear code and great test coverage.

One minor blocking change on the lockup wordmark, otherwise some non-blocking thoughts/opinions to take or leave as you see fit

**Comment by alexgibson** [comment]
@maureenlholland updated, thanks for the eagle-eyed review! 💯

File: media/css/firefox/whatsnew/whatsnew-105-eu.scss
Comment (maureenlholland) [reply media/css/firefox/whatsnew/whatsnew-105-eu.scss:102]: Scales better than I thought - nice fix!

**Review by maureenlholland — APPROVED** [review]
code r+ (I assume still waiting on final quote copy before merge?)

**Comment by alexgibson** [comment]
@maureenlholland thanks - last I heard was there may be a wordmark / logo change, so just waiting on that coming through.

**Comment by alexgibson** [comment]
Got the go-ahead to move forward with this, merging!

**Pull request merged**
